cloudgen_firewall/assets/logs/cloudgen_firewall.yaml

```diff
@@ -0,0 +1,457 @@
+# bypass-global-grok-parser-rules-checks
+id: cloudgen_firewall
+metric_id: cloudgen-firewall
+backend_only: false
+installation_sources:
+  - barracuda_cloudgen_logs
+facets:
+  - description: "Shows the component of the Barracuda CloudGen Firewall log."
+    facetType: list
+    groups:
+      - Barracuda CloudGen Firewall
+    name: component
```

Comment:
Keep capitalization consistent in all namens in the facets, as well as plural form.

```
component -> Component
IPs -> IP
process -> Process
session_status -> Session Status
```

Make sure to review this in all facet names.